calico-enterprise_versioned_docs/version-3.17/releases.json

```diff
@@ -1,259 +1,760 @@
 [
   {
-    "title": "master",
+    "title": "v3.17.0",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.29.4",
```

Comment:
This is v1.30.x, where x is still not set in stone, but most likely .0

Reply:
Set to 1.30.0.